### ISSUE TITLE
Falling off the map does not kill you, but re-spawns you in the centre of the map

### ISSUE THREAD

**Issue by ChrisD2012** [jasonsurratt/knockback-tickets#32]
**Describe the bug**
On the map with the houses and water, if you run off the edge of the map it re-spawns you,
A fun treat if you jump just as you full off you just appear back where you fell off.

**To Reproduce**
1. Go to the map with the house and water all around (Susan)
2. Runoff the map.
3. Boom you got away from the bad guys and are in the center of the map again

**Expected behavior**
Death

**Screenshots**

![20200601094644_1](https://user-images.githubusercontent.com/61132931/83365626-e2bf0400-a3ec-11ea-8f78-af8a33785588.jpg)

![20200601094648_1](https://user-images.githubusercontent.com/61132931/83365631-ed799900-a3ec-11ea-90ec-b5832a3401e9.jpg)

**Comment by jasonsurratt** [comment]
Yup. When I drown you for going underwater this shouldn't be an issue. :)
